### PR TITLE
Add ng tests to the test target

### PR DESCRIPTION
**Pull request by ray-milkey**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,10 +8,10 @@ build:
 
 test: # @HELP run the unit tests and source code validation
 test: deps build lint license_check
-    #Waiting in Issue #20  cd web/onos-gui && ng test --browsers=ChromeHeadless --watch=false
+	cd web/onos-gui && ng test --browsers=ChromeHeadless --watch=false
 
 coverage: # @HELP generate unit test coverage data
-coverage: deps build license_check
+coverage: deps build license_check test
```
Comment (SeanCondon) [inline Makefile]: maybe remove deps, build and license_check here since they are called from test anyway

File: Makefile [hunk 2]
```diff
@@ -8,10 +8,10 @@ build:
 
 test: # @HELP run the unit tests and source code validation
 test: deps build lint license_check
```
Comment (SeanCondon) [inline Makefile]: really 'build' is not required for test - the tests will run on their own if the build had not been run. I don't know how that compares to "go" Makefile structure

File: web/onos-gui/src/karma.conf.js [hunk 3]
```diff
@@ -4,6 +4,16 @@
 module.exports = function (config) {
   config.set({
     basePath: '',
+    customLaunchers: {
+      ChromeHeadlessNoSandbox: {
```
Comment (SeanCondon) [inline web/onos-gui/src/karma.conf.js]: nice

File: build/bin/make-coverage [hunk 4]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
```
Comment (SeanCondon) [inline build/bin/make-coverage]: you have to download it every time? Dockerfile doesn't allow install?

Comment (ray-milkey) [reply build/bin/make-coverage]: We use someone else's image to run this stuff. I suppose we could make our own, but I don't like the maintenance headache of maintaining these build only images.

**Pull request merged**